web_store/orders/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from orders.views import fetch_customer_orders, fetch_ordered_products_by_period, fetch_ordered_products_by_days
```

Comment:
Мне кажется, что это лишнее
```suggestion
from .views import fetch_customer_orders, fetch_ordered_products_by_period, fetch_ordered_products_by_days
```